pegasus/sites.v3/code.org/views/share_privacy.haml

```diff
@@ -0,0 +1,49 @@
+%div{style: 'position:absolute; top:5px; right:5px', id: 'share_privacy_policy'}
+  %label
+    Share this notice with parents:
```

Comment:
Do we need to loc any of this?

Reply:
Do you mean localize? The spec explicitly calls out that this should only be in English (and unfortunately we can't translate the remind button). I don't think this page (/student-privacy) has been translateable in the past, though we're changing that.

@dju90 thoughts on this?

Reply:
Sorry yes, localize.

Reply:
Somehow didn't see this previously...the privacy notice is currently a nice-to-have in terms of making it translatable so making this part localizable is included in that prioritization (which is to say, not important right now)